src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/entity/CsipRecord.kt

```diff
@@ -127,6 +127,8 @@ class CsipRecord(
   ) = apply {
     auditEvents = (auditEvents ?: mutableSetOf())
     auditEvents!!.add(auditRequest)
+    // change modifiedAt to force hibernate to save audit record
+    lastModifiedAt = LocalDateTime.now()
```

Comment:
why isn't audit record saved without this?
is that some undocumented bug or weirdness?